### PR TITLE
feat: filters

### PR DESCRIPTION
**Pull request by mertkaya1033**
# Description
This PR is a result of pair programming with @usirin and is to sync up the code.

### CODE REVIEW

File: src/features/location-filters/components/FilterService.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  useServiceFilter,
+  filters as serviceFilters,
+  ServiceCategory,
+} from "../stores/useServiceFilter";
+
+export const FilterService = () => {
+  const { t } = useTranslation("home");
+  const { isOpen, actions, selectedCategories } = useServiceFilter();
+
+  console.log({ selectedCategories });
```
Comment (eraygundogmus) [inline src/features/location-filters/components/FilterService.tsx:16]: ?

**Comment by eraygundogmus** [comment]
your pr description is not valid

**Comment by usirin** [comment]
i asked him to open a pr so i could use what he did as ref, no worries.

File: src/features/location-filters/components/FilterService.tsx (hunk 1 above)
Comment (mertkaya1033) [reply src/features/location-filters/components/FilterService.tsx:16]: It is for debugging purposes, will be cleaned up later.

Comment (eraygundogmus) [reply src/features/location-filters/components/FilterService.tsx]: > i asked him to open a pr so i could use what he did as ref, no worries.

lol draft PR is a feature for this usecase, ok

Comment (usirin) [reply src/features/location-filters/components/FilterService.tsx]: oh i keep forgetting that github has a draft feature now man @eraygundogmus sorrryyyyyy mybad.

**Pull request closed without merging**